app/presenters/route_summary_card_data_presenter.rb

```diff
@@ -79,10 +82,27 @@ def default_route_card(index)
     }
   end
 
+  def edit_secondary_skip_link
+    if Settings.features.branch_routing && all_routes.find(&:secondary_skip?).present?
+      govuk_link_to("Edit", edit_secondary_skip_path(form_id: form.id, page_id: page.id))
```

Comment:
This should use a translation string, it would be nice if we had a generic "Edit" key 🤔 

```suggestion
      govuk_link_to(t(".edit"), edit_secondary_skip_path(form_id: form.id, page_id: page.id))
```